src/styles/blocks/content/card.scss

```diff
@@ -0,0 +1,86 @@
+.card-container {
+  &__card {
+    margin: 0 24px 46px;
+    max-width: 200px;
+  }
+}
```

Comment:
move this code to file card-container.scss or the one which you would have after changes in html.